package.json

```diff
@@ -28,6 +28,7 @@
     "@zendeskgarden/react-loaders": "8.76.2",
     "@zendeskgarden/react-modals": "8.76.2",
     "@zendeskgarden/react-notifications": "8.76.2",
+    "@zendeskgarden/react-pagination": "^8.76.8",
```

Comment:
We should align the version with the other Garden packages

Reply:
Ok changed to 8.76.2